src/targets/gpu/schedule_model.cpp

```diff
@@ -77,28 +77,28 @@ MIGRAPHX_REGISTER_OP(wait_event)
 MIGRAPHX_REGISTER_OP(set_stream)
 
 std::size_t schedule_model::concurrency() const { return streams; }
-void schedule_model::sched(module& p, instruction_ref ins, std::size_t n) const
```

Comment:
The clang-format error check looks to be unrelated to the changes you made to this file, but I think you should update it anyways. We must have missed the change when we upgraded versions awhile back.